dist/src/Form/Extension/ProductTypeExtension.php

```diff
@@ -27,6 +27,7 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
             'required' => false,
             'label' => 'sylius.form.product.description',
             'locale' => $builder->getName(),
+            //'tags' => ['-map'],
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏shouldn't we keep it uncommented to test the exclusion feature?

Reply:
@delyriand Yes we could… but I prefer to see the Google Map Ui Element, but I agree with you. Maybe we should create a simple Ui Element, duplicating a default one and exclude it with the `-exclude-me` tag. WDYT?

Reply:
Yes, when I read the PR, I was thinking about creating an "empty" UI Element and excluding it. So duplicating one is a good idea!